packages/reports/app/styles/navi-reports/components/navi-date-range-picker.less

```diff
@@ -55,10 +55,9 @@
 
   &__apply-btn {
     font-size: @font-size-mid;
-    border-radius: 5px;
     border-width: 0 0 3px 0;
-    padding: 2px 13px;
-    margin: 0 0 4px 10px;
+    margin: 0 0 0 10px;
```

Comment:
`margin-left: 10px;` ?

Reply:
@mattgraham1995 maybe with `margin: 0` to not unset the other margins

Reply:
Yeah I also wondered that, but we checked in the existing version and there is no conflicting margin.